docs/design/gang-podgroup.md

```diff
@@ -0,0 +1,128 @@
+# Gang PodGroups
+
+[@umiaplha](http://github.com/umialpha); Feb 20, 2020
+
+## Motivation
+Pods are grouped by PodGroup, but what if we need to group `PodGroups`?
+For example, Job1{taskNum: 3, MinMember: 2}, Job2{taskNum:3, MinMember:2}, Job1 must co-run with Job2.
+Currently, we cannot handle such case.
+Thus, a new feature named `Gang PodGroups` is introduced.
+
+## Function Detail
+A type of string "SubGroup" is introduced into the PodGroupSpec:
+```go
+type PodGroupSpec struct {
+	// SubGroups defines the groups of PodGroup.
+	// When PodGroups grouped together, they are scheduled as a whole.
+	SubGroup string
```

Comment:
Prefer to add `SubGroupSpec` for its attributes, e.g. `minMember`. It's complex to reconstruct PodGroup and its SubGroup in scheduler.

Reply:
> Prefer to add `SubGroupSpec` for its attributes, e.g. `minMember`. It's complex to reconstruct PodGroup and its SubGroup in scheduler.

 `SubGroup` is  just like `Namespace` or `Queue`, specifying the SubGroup Name it belongs.

I have completed the code, shall I send a new PR for the detail?
Or we can talk more about your idea, maybe I misunderstand your idea.

Reply:
let's add an item to volcano weekly meeting or book another meeting for that :)

Reply:
> let's add an item to volcano weekly meeting or book another meeting for that :)

Great, what is the time and link about the meeting this week? I would like to audit :)

Reply:
There's a weekly meeting at 14:30, every Thursday (Beijing Time), and here's the agenda of Volcano weekly meeting: https://docs.google.com/document/d/1YLbF8zjZBiR9PbXQPB22iuc_L0Oui5A1lddVfRnZrqs/edit?usp=sharing